site/content/docs/5.3/customize/sass.md

```diff
@@ -85,6 +87,45 @@ In your `custom.scss`, you'll import Bootstrap's source Sass files. You have two
 
 With that setup in place, you can begin to modify any of the Sass variables and maps in your `custom.scss`. You can also start to add parts of Bootstrap under the `// Optional` section as needed. We suggest using the full import stack from our `bootstrap.scss` file as your starting point.
 
+## Compiling
+
+In order to use your custom Sass code as CSS in the browser, you need a Sass compiler. Sass ships as a CLI package, but you can also compile it with other build tools like [Gulp](https://gulpjs.com/) or [Webpack](https://webpack.js.org/), or with a GUI application like [CodeKit](https://codekitapp.com/) or [Prepros](https://prepros.io/). Some IDEs also have Sass compilers built in or as downloadable extensions.
```

Comment:
I think it's good to provide examples but CodeKit and Prepros seem to be paid products, so I'm not sure it is our "role" to emphasize some paid tools. I don't have a strong opinion here or the answer, I just got doubts on my side in order to be fair with all products out there :)